lms/templates/bookmarks/bookmarks_button.underscore

```diff
@@ -0,0 +1,5 @@
+<label for="course-bookmarks-button" class="sr"><%= gettext("Course") %></label>
+<button type="button" class="bookmarks-button" id="course-bookmarks-button">
```

Comment:
@muhammad-ammar, I also noticed that the button functions as a toggle (clicking it again, returns the main content to its previous state.  If you want to do that, you need to add an ```aria-pressed``` attribute to the button element.  The value is either true or false.  You will need to maintain the correct state if content is changed by any other action.